.kokoro/dashboard.sh

```diff
@@ -26,3 +26,19 @@ cd dashboard/
 echo -e "\n******************** BUILDING THE DASHBOARD ********************"
 
 mvn --fail-at-end clean install
+INSTALL_RETURN_CODE=$?
+RETURN_CODE=${INSTALL_RETURN_CODE}
+
+case ${JOB_TYPE} in
+converge)
+    mvn exec:java -Dexec.args="-f ../pom.xml --report"
+    CONVERGE_RETURN_CODE=$?
+    if [[ $INSTALL_RETURN_CODE -eq 0 ]]
+    then
+      RETURN_CODE=${CONVERGE_RETURN_CODE}
+    fi
+    ;;
+esac
+
+echo "exiting with ${RETURN_CODE}"
```

Comment:
This would only tell you if the report was built successfully or not (which is important) but I'm not seeing the output being checked.

Ideally, if we have full convergence, the script should exit with 0 and output `Shared dependencies converge \o/`. Otherwise, it should exit with error code 1 and a list of clients that are not converging (which is already being output by the report).